.travis.yml

```diff
@@ -11,10 +11,14 @@ script:
   - npm test
 services:
   - postgresql
+env:
+- NODE_ENV=test
 cache: 
   - npm install
 before_script:
   - npm install sequelize-cli@2.8.0 -g
```

Comment:
Not a blocker but putting the flags before the actual  packages to be installed would be better i.e. If you needed to install several packages this would be `npm install -flag(s) pkg1 pkg2 pkg3`